### ISSUE TITLE
[4b6] Selecting image to insert gives popover of fields to complete, but just looks plain wrong

### ISSUE THREAD

**Issue by PhilETaylor** [joomla/joomla-cms#31909]
### Steps to reproduce the issue

Joomla 4 beta 6

Edit Article (CodeMirror or TinyMCE editor) 
use CMS Content or Image button to load Media Manager Modal
Click and image 

### Expected result

Im unsure what the aim was... but certainly not this.

### Actual result

Watch:

https://user-images.githubusercontent.com/400092/104050095-f5e54e00-51dd-11eb-94b1-e42edd2c67f6.mp4

Im guessing you now want me to fill in this information and then click insert image as normal? 

This just seems half implemented, and in totally the wrong place - the popover looks out of place and at a minimum needs to have some padding around the fields to the right side

This is Joomla 4 where media manager is a flagship feature - we can do better than this :)

**Comment by Varad-VK** [comment]
please assign this to me and guide me as i am new to open source

**Comment by infograf768** [comment]
adding `padding-inline-end: 1rem;` to `joomla-field-mediamore details {` in `.../build/media_source/system/css/fields/joomla-image-select.css` would solve the padding issue both in LTR and RTL.

**Comment by infograf768** [comment]
Change `.../build/media_source/system/css/fields/joomla-image-select.css` 
from
```
joomla-field-mediamore details {
  position: absolute;
  background: #ffffff;
}
```
to

```
joomla-field-mediamore details {
  position: absolute;
  background: #f5f5f5;
  padding: 0.5rem;
}
```
to get

<img width="715" alt="Screen Shot 2021-03-13 at 10 51 02" src="https://user-images.githubusercontent.com/869724/111026352-0e463480-83ea-11eb-935c-e95b0fc59883.png">

**Comment by richard67** [comment]
Closing as it has been solved with PR #32728 . Feel free to re-open if something is left to be done.
